packages/arb-token-bridge-ui/src/tokenLists.tsx

```diff
@@ -152,15 +152,18 @@ export function fetchTokenLists(): Promise<void> {
       const tokenListsWithBridgeTokenListId = responses
         .filter(({ isValid }) => isValid)
         // Attach the bridge token list id so we can easily retrieve a list later
-        .map(({ data }, index) => {
-          const token = BRIDGE_TOKEN_LISTS[index]
-          if (!token) {
+        .map(({ data }) => {
+          const tokenList =
+            data &&
+            BRIDGE_TOKEN_LISTS.find(list => list.name.includes(data.name))
```

Comment:
I have 2 small concerns - 
1. This is iterating the BRIDGE_TOKEN_LISTS for each token in the response, is it possible to optimize it?
2. Instead of running a check on the `name`, is there any stricter check we can implement, probably on the id?